dripconfig/helpers.py

```diff
@@ -34,3 +39,48 @@ class SysLogHandler(_SysLogHandler):
     def emit(self, record):
         record.ident = os.path.basename(sys.argv[0])
         super(SysLogHandler, self).emit(record)
+
+
+class StatsdHandler(DatagramHandler):
+
+    def __init__(self, host, port):
+        super(StatsdHandler, self).__init__(host, port)
+
+    def _metric(self, record):
+        return 'errors.{}.{}.{}'.format(
+            record.name, record.levelname, record.funcName
+        )
+
+    def _prepare_increment(self, metric):
+        return '{}:1|c'.format(metric)
+
+    def emit(self, record):
+        """
+        Increment a statsd counter for the error that occurred.
+        """
+        try:
+            s = self._prepare_increment(self._metric(record))
+            self.send(s)
+        except (KeyboardInterrupt, SystemExit):
+            raise
+        except:
+            self.handleError(record)
+
+
+class StatsdErrorFilter(Filter):
+    """
+    This filter ensures that only specific errors are reported to Graphite.
+    """
+
+    WHITELIST = [
+        ('kafka.consumer.fetcher', 'ERROR', '_call_backs'),
+        ('kafka.coordinator.consumer', 'ERROR', 'commit_offsets_sync'),
+        ('kafka.coordinator', 'ERROR', '__call__'),
+        ('kafka.coordinator', 'ERROR', '_failed_request'),
+        ('kafka.conn', 'ERROR', 'recv'),
```

Comment:
dripconfig isn't exactly widely adopted, but it is open source, so I don't think a whitelist specific to our use case is appropriate... I don't know much about filters in Python logging but could be there some way to pass the whitelist in from the config dict at configuration time?